common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -265,45 +265,48 @@ public class CommonValue {
                         + MLModel.LAST_UNDEPLOYED_TIME_FIELD
                         + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
                         + "      \""
+                        + MLModel.MODEL_INTERFACE_FIELD
+                        + "\" : {\"type\": \"flat_object\"},\n"
+                        + "      \""
                         + MLModel.GUARDRAILS_FIELD
                         + "\" : {\n" +
-                "          \"properties\": {\n" +
-                "            \"input_guardrail\": {\n" +
-                "              \"properties\": {\n" +
-                "                \"regex\": {\n" +
-                "                  \"type\": \"text\"\n" +
-                "                },\n" +
-                "                \"stop_words\": {\n" +
-                "                  \"properties\": {\n" +
-                "                    \"index_name\": {\n" +
-                "                      \"type\": \"text\"\n" +
-                "                    },\n" +
-                "                    \"source_fields\": {\n" +
-                "                      \"type\": \"text\"\n" +
-                "                    }\n" +
-                "                  }\n" +
-                "                }\n" +
-                "              }\n" +
-                "            },\n" +
-                "            \"output_guardrail\": {\n" +
-                "              \"properties\": {\n" +
-                "                \"regex\": {\n" +
-                "                  \"type\": \"text\"\n" +
-                "                },\n" +
-                "                \"stop_words\": {\n" +
-                "                  \"properties\": {\n" +
-                "                    \"index_name\": {\n" +
-                "                      \"type\": \"text\"\n" +
-                "                    },\n" +
-                "                    \"source_fields\": {\n" +
-                "                      \"type\": \"text\"\n" +
-                "                    }\n" +
-                "                  }\n" +
-                "                }\n" +
-                "              }\n" +
-                "            }\n" +
-                "          }\n" +
-                "        },\n"
+                        "          \"properties\": {\n" +
```

Comment:
not a big problem , but better to make the code consistent. The lines above this line has `+` in the beginning of line, can we keep it consistent? It's fine to fix in a new PR if no enough bandwidth.

Reply:
Looks like this part has different indention, please run `./gradlew spotlessApply` to format code. Also, I suggest we can move these configurations to a resource file with json suffix, it would be much easier to read and maintain.